homeassistant/components/webostv/media_player.py

```diff
@@ -206,6 +187,20 @@ def update_sources(self):
             ):
                 self._source_list[source["label"]] = source
 
+        # special handling of live tv since this might not appear in the app or input lists in some cases
```

Comment:
Can't this handling be done in the library?

Reply:
The library doesn't have a concept of "sources", but just apps and inputs, I'll think about whether it could make sense to add this to the list of apps rather.

Reply:
So thinking about it further, I would prefer to leave this workaround in home assistant.  It's basically not needed for standalone usage of the client library, since a user could always call launch_app("com.webos.app.livetv") directly, and the state will already correctly propagate to client.current_appId.

It's needed here basically to work around the way in which this functionality is mapped onto source handling in the home assistant media_player api and standard UI implementation.  (ie if the source doesn't appear in entity.source_list then both entity.source and select_source are problematic)

Reply:
Ok.